### PR TITLE
Hw5

### PR DESCRIPTION
**Pull request by AnnaTsukiyo**
Added Hometask5 (POJO Order)

### CODE REVIEW

File: src/main/java/com/savchenko/homeworks/hometask5/Order.java [hunk 1]
```diff
@@ -1,4 +1,64 @@
 package com.savchenko.homeworks.hometask5;
 
+import java.util.Date;
+
 public class Order {
+    private String item;
+    private double size;
+    private int price;
```
Comment (Eugene-Berezhnyi) [inline src/main/java/com/savchenko/homeworks/hometask5/Order.java]: price as int? Really?

File: src/main/java/com/savchenko/homeworks/hometask5/Order.java [hunk 2]
```diff
@@ -1,4 +1,64 @@
...
+    public int setPrice() {
+        return price;
+    }
+
+    public Date setDeliveryDate(String date) {
+        return deliveryDate;
+    }
+
+    public String getItem() {
+
```
Comment (Eugene-Berezhnyi) [inline src/main/java/com/savchenko/homeworks/hometask5/Order.java]: you do not need this empty line

File: src/main/java/com/savchenko/homeworks/hometask5/Order.java [hunk 3]
```diff
@@ -1,4 +1,64 @@
 package com.savchenko.homeworks.hometask5;
 
+import java.util.Date;
```
Comment (Eugene-Berezhnyi) [inline src/main/java/com/savchenko/homeworks/hometask5/Order.java:3]: Override equals and hashcode

**Comment by Eugene-Berezhnyi** [comment]
Мерж

**Comment by AnnaTsukiyo** [comment]
Occasionally closed the pull request instead of merging, please ignore my pull request (nothing changed in my branch)

**Pull request merged**